src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,25 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { Todo, User } from '../../types/types';
+import usersFromServer from '../../api/users';
```

Comment:
Avoid importing `usersFromServer` directly in this component. According to the checklist, data should be prepared in the `App` component and passed down to child components as props.